npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
<img width="195" alt="screen shot 2017-02-17 at 9 35 32 am" src="https://cloud.githubusercontent.com/assets/15853081/23074060/df6e33c0-f4f4-11e6-9bb9-952c99085ac4.png">

@ChelseaSkovgaard It might have to do with the current file structure. Can you try moving the .gitignore to the root of the repo and then push it up again?